layouts/partials/footer.html

```diff
@@ -1,24 +1,26 @@
-<footer class="footer">
-  <section class="container">
-    {{ with .Site.Params.footercontent | safeHTML }}
-      <p>{{.}}</p>
-    {{ end }}
-    {{ if not .Site.Params.hideCopyright }}
-      {{ with .Site.Params.since }}
-        © {{ if lt . now.Year }}{{ . }} - {{ end }}{{ now.Year }}
-      {{ else }}
-        © {{ now.Year }}
+{{ if .Site.Params.footer }}
```

Comment:
```suggestion
{{ if .Site.Params.footer | default true }}
```
Footer should be displayed by default.